api/src/main/java/org/apache/cloudstack/api/command/admin/offering/CreateServiceOfferingCmd.java

```diff
@@ -433,6 +437,10 @@ public Long getStoragePolicy() {
         return storagePolicy;
     }
 
+    public Boolean getDynamicScalingEnabled() {
+        return isDynamicScalingEnabled == null ? Boolean.TRUE : isDynamicScalingEnabled;
```

Comment:
fixed all Boolean to boolean, thanks for that.
Yes all old service offerings prior to upgrade will be marked as dynamically scalable. This is made not to break existing VMs because service offering was not part of dynamic scaling decision before. After upgrade we need to bypass that check for old VMs, which can be achieved by marking them true.

Reply:
Copy, thanks